Integrations/EWSMailSender/EWSMailSender.py

```diff
@@ -126,6 +129,14 @@ def send_email(to, subject, body="", bcc=None, cc=None, replyTo=None, htmlBody=N
         if len(file_entries_for_attachments) != len(attachments_names):
             raise Exception("attachIDs and attachNames lists should be the same length")
 
+    if attachCIDs:
+        file_entries_for_attachments_inline = attachCIDs.split(",")
+        for att_id_inline in file_entries_for_attachments_inline:
+            att_name_inline = demisto.getFilePath(att_id_inline)['name']
+            if isinstance(att_name_inline, list):
```

Comment:
when did you ever got a list from getFilePath?

Reply:
Don't think that it is necessary but tried to save the same convention as in previous block of code. I don't want to change the previous block of code.

Reply:
weird... but seems legit